AmazonChimeSDKDemo/AmazonChimeSDKDemo.xcodeproj/xcshareddata/xcschemes/AmazonChimeSDKDemoBroadcastSPM.xcscheme

```diff
@@ -48,6 +48,7 @@
       savedToolIdentifier = ""
       useCustomWorkingDirectory = "NO"
       debugDocumentVersioning = "YES"
+      askForAppToLaunch = "Yes"
```

Comment:
is the change related to reconnect timeout config?

Reply:
> is the change related to reconnect timeout config?

No, it's Xcode auto update.